macros/edr/metadata_collection/get_columns_from_information_schema.sql

```diff
@@ -1,21 +1,35 @@
-{% macro get_columns_from_information_schema(full_schema_name) %}
-    {{ return(adapter.dispatch('get_columns_from_information_schema', 'elementary')(full_schema_name)) }}
+{% macro get_columns_from_information_schema(schema_array) %}
+    {{ return(adapter.dispatch('get_columns_from_information_schema', 'elementary')(schema_array)) }}
 {% endmacro %}
 
-{# Snowflake and Bigquery #}
-{% macro default__get_columns_from_information_schema(full_schema_name) %}
-    {% set full_schema_name_split = full_schema_name.split('.') %}
-    {% set database_name = full_schema_name_split[0] %}
-    {% set schema_name = full_schema_name_split[1] %}
+{# Snowflake, Bigquery#}
+{% macro default__get_columns_from_information_schema(schema_array) %}
+    {%- set database_name, schema_name, schema_relation = schema_array %}
```

Comment:
You don't need to get the schema_relation, you can get only the db and schema name -
database_name, schema_name = schema_tupple
Then create the relation using api.Relation.create (as written in get_configured_schemas_from_graph)